design/observers.md

```diff
@@ -0,0 +1,211 @@
+As of now we have Validator Nodes only responsible for write and read requests as well as catching up other Nodes and clients.
+With a state proof support, we can create additional rings of Observer Nodes that can process read requests and reduce load on Validator nodes.
+
+## Goals (in priority order)
+
+- Goal1: Reduce Read Load
+    - Reduce read load on Validators
+
+- Goal2: More Nodes
+    - Have a big number of Nodes available for read requests (we have an essential limit on the size of Validator Nodes because of how chatty RBFT is)
+    
+- Goal3: Reduce Catch-up Load
+    - Reduce catch-up load on Validators
+    
+- Goal4: Reduce Write Load
+    - Be able to send write requests to one Node only (either Validator or Observer)
+    
+- Goal5: Promotion
+    - Be able to replace suspicious Validators to Gatekeepers (promote Gatekeepers)
+    
+- Goal6: Untrusted Observers
+    - Be able to have unlimited number of Nodes with a copy of the ledger
+    
+- Goal7: Fast Catch-up of clients
+    - Be able to bootstrap catch-up for clients using one Node (either Validator or Observer)
+    
+- Goal8: Fast Catch-up of Nodes 
+    - Be able to bootstrap catch-up for Nodes using one Node (either Validator or Observer)
```

Comment:
This is a goal from the use cases point of view: we need to be able to perform catch-up as fast as possible for Nodes in the pool.

Technically it can be done the same way as in previous Goal.